charts/app/templates/metabase/deployment.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "metabase.fullname" . }}
+  labels:
+    {{- include "metabase.labels" . | nindent 4 }}
+spec:
+  strategy:
+    type: {{ .Values.metabase.deploymentStrategy }}
+  {{- if not .Values.metabase.autoscaling.enabled }}
+  replicas: {{ .Values.metabase.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "metabase.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.metabase.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "metabase.selectorLabels" . | nindent 8 }}
+    spec:
+      securityContext:
+        {{- toYaml .Values.metabase.podSecurityContext | nindent 8 }}
+      containers:
```

Comment:
As per the sonar warning, Are we able to put a request / reservation limit on this container to ensure that we have sufficient quota to spin up the container?

Reply:
tested this requested change - as suspected we cannot specify ephemeral storage for pods as we do not control the nodes. So this sonar issue is effectively a wontfix.